web/html/src/manager/virtualization/pools/list/pools-list.js

```diff
@@ -0,0 +1,204 @@
+// @flow
+import * as React from 'react';
+import { Tree } from 'components/tree/tree';
+import type { TreeItem, TreeData } from 'components/tree/tree';
+import { CustomDiv } from 'components/custom-objects';
+import { ProgressBar } from 'components/progressbar';
+import { DataHandler } from 'components/data-handler';
+import { SearchField } from 'components/data-handler';
+import { VirtualizationPoolsListRefreshApi } from '../virtualization-pools-list-refresh-api';
+
+declare function t(msg: string): string;
```

Comment:
we don't need this declare t() anymore. It's already included in the flow globals.js file

Reply:
cool! I'll remove it then